src/TypeSpec.Extension/Emitter.Csharp/src/sdk-context-options.ts

```diff
@@ -7,6 +7,7 @@ export const azureSDKContextOptions: CreateSdkContextOptions = {
     versioning: {},
     additionalDecorators: [
         "Azure\\.ClientGenerator\\.Core\\.@useSystemTextJsonConverter",
+        "Azure\\.ClientGenerator\\.Core\\.@deserializeEmptyStringAsNull",
         "Azure\\.ResourceManager\\.@armResourceOperations",
```

Comment:
I feel like we should delete the existing "models-to-treat-empty-string-as-null" in options.ts.

Reply:
We could do it and know if it impacts any real RPs in our regen.
If it impacts anything, we could do it in a separate PR.